src/asciidoctorWebViewConverter.ts

```diff
@@ -40,6 +121,64 @@ export class AsciidoctorWebViewConverter {
    */
   convert (node, transform) {
     const nodeName = transform || node.getNodeName()
+    if (nodeName === 'document') {
+      // Content Security Policy
+      const nonce = new Date().getTime() + '' + new Date().getMilliseconds()
+      const csp = getCspForResource(this.securityLevel, nonce)
+      const content = node.getContent()
+      const syntaxHighlighter = node.$syntax_highlighter()
+      const linkcss = node.isAttribute('linkcss')
+      let assetUriScheme = node.getAttribute('asset-uri-scheme', 'https')
+      if (assetUriScheme.trim() !== '') {
+        assetUriScheme = `${assetUriScheme}:`
+      }
+      // REMIND: we should not use cdnjs in the WebView!
+      const cdnBaseUrl = `${assetUriScheme}//cdnjs.cloudflare.com/ajax/libs`
+      if (syntaxHighlighter !== Opal.nil) {
+        if (syntaxHighlighter['$docinfo?']('head')) {
+          const opts = {
+            cdn_base_url: cdnBaseUrl,
+            linkcss: linkcss,
+            self_closing_tag_slash: '/',
+          }
+          syntaxHighlighter.$docinfo('head', node, Opal.hash2(Object.keys(opts), opts))
+        }
+      }
+      // QUESTION: should we embed font awesome?
+      // QUESTION: should we embed the following fonts (when using the default stylesheet)?
+      // Open+Sans:300,300italic,400,400italic,600,600italic
+      // Noto+Serif:400,400italic,700,700italic
+      // Droid+Sans+Mono:400,700
```

Comment:
Yes, I think so. I guess this extension should be functional without internet access.

Reply:
I've added Noto Serif, Open Sans and Roboto mono